k8s/common/sds.yaml

```diff
@@ -1,5 +0,0 @@
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: sds
```

Comment:
probably safe to delete, i've never seen this before but worth checking if there's anything in there

Reply:
Empty in the lower envs except for dev where it has a failed plum install.   Will leave for now and probably best to revisit in a later PR.

Reply:
Branch now rolled back to previous approved commit.